data/transition-sites/gds_career.yml

```
@@ -0,0 +1,6 @@
+---
+site: gds_career
+whitehall_slug: government-digital-service
+homepage: https://www.gov.uk/government/organisations/government-digital-service
+tna_timestamp: 20201010101010 # Stub timestamp - site is not in TNA
+host: www.civil-service-careers.gov.uk
```

Looking at the ticket, shouldn't the host be `gdscareers.gov.uk`?

```suggestion
host: gdscareers.gov.uk
```

---

docs lead me to believe that it was "the new homepage for the old site" but I'll admit they're not especially clear and I could be reading it wrong 

https://github.com/alphagov/transition-config/blob/main/README.md

---

I thought we configured the old domain here, then set up the new URL in the transition UI?

---

Or set the new URL as the `homepage` value?

---

Just checked some of the others, you're 100% right, will amend!

---

done!